GUI fixes (#676)
vrc-get/vrc-get: CHANGELOG-gui.md

## [Unreleased]
### Added
- Japanese Localization `#674`

### Changed
- Package names on the Apply Changes dialog are now bold `#676`

> - Deleting a project folder moves the project folder to trach instead of hard deleting `#676`

Reviewer: ```suggestion
- Deleting a project now moves its folder to trash instead of hard deleting `#676`
```

### Deprecated

### Removed

### Fixed
- World icon does not appear in the project list `#625`

> - Remove button on the manage project page is not working `#676`

Reviewer: ```suggestion
- Remove project button on the manage project page is not working `#676`
```

## [Unreleased]
### Added
- Japanese Localization `#674`

### Changed

> - Package names on the Apply Changes dialog are now bold `#676`

Reviewer: ```suggestion
- Package names on the Apply Changes dialog and a few other texts are now bold `#676`
```

- Deleting a project now moves its folder to trash instead of hard deleting `#676`

### Deprecated

### Removed

### Fixed
- World icon does not appear in the project list `#625`
- Remove project button on the manage project page is not working `#676`
- Project name warning is too bright `#676`
- Unable to touch any buttons if Apply Changes dialog is long `#676`
- The package order is not deterministic `#676`
- Path separator is not correct on windows `#676`
- Same project can be added multiple times `#676`
- Create button on the creating project dialog is not disabled `#676`
- We can control the web ui while file picker is open `#676`
- Unrecoverable error when trying to add a non-project folder `#676`
- Prerelease packages can be installed if version range has prerelease while the option is off `#676`
- Handling whitespaces in the path of the project is not correct `#676`

> - We could not add a unity manually `#676`

Reviewer: ```suggestion
- We could not add unity manually `#676`
```
付けるならanですが、まあ無くても良いかなって…

```suggestion
- We could not add an unity manually `#676`
```

Author: 調べた感じ半母音はaですねそして発音は /júːnəṭi/ で /j/ は半母音なので a で良さそう
